src/connectedk8s/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+1.1.11
```

Comment:
This needs to change to 1.1.12 as we recently released the HELM client fix with 1.1.11

Reply:
Please merge with latest main branch

Reply:
merged latest main.